plugin/src/main/java/org/wildfly/plugin/provision/AbstractProvisionServerMojo.java

```diff
@@ -178,7 +177,7 @@ abstract class AbstractProvisionServerMojo extends AbstractMojo {
     String layersConfigurationFileName;
 
     @Parameter(alias = "channels", required = false)
-    List<ChannelCoordinate> channels;
+    List<ChannelConfiguration> channels;
```

Comment:
What are the differences here? Are there parameter name differences? If so we'll need to change the version to something like 4.1.0 before this can be merged.

Reply:
@jamezp , yes we have an incompatible change there, we introduce a sub element `manifest-coordinate` inside a Channel. +1 for 4.1.0, I was going to suggest it.